src/components/editCenter/Description.js

```diff
@@ -52,6 +53,7 @@ const Description = ({step, setStep, result, setResult, clubs, setClubs}) => {
         updateCenter(result.id,result).then(response => {
             if(response.status && response.status ===  400 && response.data.message){
```

Comment:
Remove catch from updateCenter service function, so you can catch any error here, intead of response.status ===  400